tox.ini

```diff
@@ -35,7 +35,7 @@ deps =
 # default tox env excludes integration and serial tests
 commands =
     # These need to be installed in a specific order
-    pip install urllib3==1.22
+    pip install urllib3==1.25.7
```

Comment:
This is to surpress the 3.8 warnings:

> DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    class AuthContext(collections.Mapping):